src/scripts.js

```diff
@@ -139,11 +146,31 @@ function postTripRequest(){
   })
 }
 
-//eventListeners
+function checkLoginCredentials(){
+  let formattedUsername = username.value.slice(0,8)
+  let usernameID
+  if(username.value.length === 10){
+    usernameID = Number(username.value.slice(-2))
+  } else if(username.value.length === 9){
```

Comment:
I thought slice took two arguement? index[0] how many to slice. interesting if this worked!